lib/net/ldap.rb

```diff
@@ -609,13 +609,7 @@ def authenticate(username, password)
   def encryption(args)
     warn "Deprecation warning: please give :encryption option as a Hash to Net::LDAP.new"
     return if args.nil?
```

Comment:
That's a great point. :+1: to leaving this. Especially since it's a deprecated method.